tools/devtool

```diff
@@ -412,6 +414,29 @@ cmd_help() {
     echo "            --performance            Tweak various setting of the host running the tests (such as C- and P-states)"
     echo "                                     to achieve consistent performance. Used for running performance tests in CI."
     echo ""
+    echo "    build_ci_artifacts [all|rootfs|kernels]"
+    echo "        Builds the rootfs and guest kernel artifacts we use for our CI."
+    echo "        The following sub-commands are accepted:"
+    echo ""
+    echo "            all (default)"
+    echo "                Build CI rootfs and default guest kernels using configurations from"
+    echo "                resources/guest_configs."
+    echo "                This will patch the guest configurations with all the patches under"
+    echo "                resources/guest_configs/patches."
+    echo "                This is the default command, if no command is chosen."
+    echo ""    
+    echo "            rootfs"
+    echo "                Builds only the CI rootfs."
+    echo ""    
+    echo "            kernels [version]"
+    echo "                Builds our the currently supported CI kernels."
+    echo ""    
+    echo "                version: Optionally choose a kernel version to build. Supported"
+    echo "                         versions are: 5.10, 5.10-no-acpi or 6.1."
+    echo ""    
+    echo "            help"
+    echo "                Displays the help message and exits."
+    echo ""    
```

Comment:
Would this work to avoid duplication?

```
cmd_build_ci_artifacts --help
```

Reply:
You mean like running it in place and getting the output? I don't think so, but I can mention here, something like:

```
    echo "    build_ci_artifacts [all|rootfs|kernels]"
    echo "        Builds the rootfs and guest kernel artifacts we use for our CI."
    echo "        Run './tools/devtool build_ci_artifacts help' for more details on the available commands"
```

Would that work?

Reply:
To clarify, I don't think it's easy to call the command here, mainly because it has extra lines (it's not an exact copy) and the indentation is different.

Reply:
I went with the approach I suggested here. Let me know if that works for you.

Reply:
Sounds good!